components/pokemonapp.brs

```diff
@@ -36,7 +37,22 @@ function onPokemonFocus(event as Object)
     updateText(selectedPokemon)
 end function
 
+function onPokemonSelect(event as Object)
+    data = event.GetData()
+    pokemonIndex = data[1]
```

Comment:
Don't use magic numbers.
https://refactoring.guru/replace-magic-number-with-symbolic-constant - check out this lesson. It will help